tendrl/node_agent/message/logger.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+
+LOG = logging.getLogger(__name__)
+
+
+class Logger(object):
+    def __init__(self, message , etcd_orm):
+        log_message = ""
+        self.message = message
+        self.etcd_orm = etcd_orm
+        if "job_id" in message.payload:
+            """ If job_id is present then
+
+            it is considered as operation
+            """
+            log_message = self.push_operation()
+        else:
+            log_message = message.payload["message"]
+        self.push_messages()
+        self._logger(log_message)
+
+    def push_operation(self):
+        self.etcd_orm.client.write(
+            self.message.payload["job_id"],
+            self.message.__dict__,
+            append=True)
+        log_message = ("%s:%s") % (
```

Comment:
@shtripat @brainfunked @r0h4n @nnDarshan @anmolbabu @nthomas-redhat 
operation updates are required some sequence key, so we cant use etcdobj save method, and message is an object but is in common so direct write to etcd is used.